ts/screens/profile/PrivacyMainScreen.tsx

```diff
@@ -1,73 +1,270 @@
+type Props = NavigationScreenProps &
+  ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
 
-type OwnProps = Readonly<{
-  navigation: NavigationScreenProp<NavigationState>;
+type State = Readonly<{
+  isAlertShown: boolean;
+  hasNewRequest: {
+    DOWNLOAD: boolean;
+    DELETE: boolean;
+  };
 }>;
 
-type Props = OwnProps;
-
 const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
-  title: "profile.main.privacy.contextualHelpTitle",
-  body: "profile.main.privacy.contextualHelpContent"
+  title: "profile.main.privacy.privacyPolicy.contextualHelpTitle",
+  body: "profile.main.privacy.privacyPolicy.contextualHelpContent"
 };
 
-/**
- * A component to show the main screen of the Privacy section
- */
-export const PrivacyMainScreen: React.SFC<Props> = props => (
-  <TopScreenComponent
-    goBack={() => props.navigation.goBack()}
-    contextualHelpMarkdown={contextualHelpMarkdown}
-    title={I18n.t("profile.main.screenTitle")}
-  >
-    <ScreenContent
-      title={I18n.t("profile.main.mainPrivacy.screenTitle")}
-      subtitle={I18n.t("profile.main.mainPrivacy.screenSubtitle")}
-    >
-      <List withContentLateralPadding={true}>
-        {/* Privacy Policy*/}
-        <ListItemComponent
-          title={I18n.t("profile.main.mainPrivacy.privacyPolicy.title")}
-          subTitle={I18n.t(
-            "profile.main.mainPrivacy.privacyPolicy.description"
-          )}
-          onPress={() => props.navigation.navigate(ROUTES.PROFILE_PRIVACY)}
-          useExtendedSubTitle={true}
-        />
-
-        {/* Remove account */}
-        <ListItemComponent
-          title={I18n.t("profile.main.mainPrivacy.removeAccount.title")}
-          subTitle={I18n.t(
-            "profile.main.mainPrivacy.removeAccount.description"
-          )}
-          onPress={unavailableAlert}
-          useExtendedSubTitle={true}
-        />
-
-        {/* Export your data */}
-        <ListItemComponent
-          title={I18n.t("profile.main.mainPrivacy.exportData.title")}
-          subTitle={I18n.t("profile.main.mainPrivacy.exportData.description")}
-          onPress={unavailableAlert}
-          useExtendedSubTitle={true}
-        />
-
-        <EdgeBorderComponent />
-      </List>
-    </ScreenContent>
-  </TopScreenComponent>
-);
+class PrivacyMainScreen extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isAlertShown: false,
+      hasNewRequest: {
+        DELETE: false,
+        DOWNLOAD: false
+      }
+    };
+  }
+
+  // Show an alert reporting the request has been submittd
+  private handleUserDataRequestAlert = (
+    choice: UserDataProcessingChoiceEnum
+  ) => {
+    const requestState = this.props.userDataProcessing[choice];
+    if (
+      pot.isSome(requestState) &&
+      (requestState.value === undefined ||
+        requestState.value.status === UserDataProcessingStatusEnum.CLOSED)
+    ) {
+      const title = {
```

Comment:
Could title and subtitle moved outside of class body?
In this way they will be create only once